content/zh/docs/tasks/security/authorization/authz-ingress/test.sh

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env bash
```

Comment:
@ericvn do we need copy the test to other language? I think those should verified in en version.